src/test/kotlin/com/zepben/evolve/services/network/testdata/ManyFeedersToOneLvFeederNetwork.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.services.network.testdata
+
+import com.zepben.evolve.cim.iec61970.base.core.BaseVoltage
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.testing.TestNetworkBuilder
+
+object ManyFeedersToOneLvFeederNetwork {
+
+    //
+    //      c1     c2         c4
+    // b0 ======+====== tx3 ------
+    //          #
+    //       c6 #
+    //          #
+    //          b5
+    //
+    // fdr7 head terminal is b0-t2
+    // fdr8 head terminal is b5-t2
+    // lvf9 head terminal is tx3-t2
+    //
+    fun create(): NetworkService {
```

Comment:
suggestion: I am assuming this is used just to test that the head terminal determines which feeders an `LvFeeder` is associated with, so I am pretty sure the minimum network would be:
```
TestNetworkBuilder()
    .fromBreaker() // b0
    .addFeeder("b0") // fdr1
    .addFeeder("b0") // fdr2
    .addLvFeeder("b0") // lvf3
```